pkgs/development/libraries/libxmlb/default.nix

```diff
@@ -28,6 +68,12 @@ stdenv.mkDerivation rec {
 
   doCheck = true;
 
+  passthru = {
+    tests = {
+      installed-tests = nixosTests.libxmlb;
```

Comment:
Is the precedent for this not on hold anymore?

Reply:
No idea, just wanted to try it out.

Reply:
Checked, guess it's not :+1: